tests/engine/processes/calcjobs/test_calc_job.py

```diff
@@ -509,6 +509,15 @@ def test_get_hash(self, get_calcjob_builder):
         _, node = launch.run_get_node(builder)
         assert node.base.extras.get(node.base.caching._HASH_EXTRA_KEY) == node.base.caching.get_hash()
 
+    def test_get_objects_attributes(self, get_calcjob_builder):
+        """Test that :meth:`aiida.orm.CalcJobNode._get_objects_to_hash` returns the expected objects."""
+        builder = get_calcjob_builder()
+        _, node = launch.run_get_node(builder)
+        objects = node.base.caching.get_objects_to_hash()  # pylint: disable=protected-access
+
+        assert 'version' not in objects
```

Comment:
This is a bit of a proxy test. Might be nice to have another test that would specifically check that the hash is independent of aiida-core version.

Reply:
Done